src/connectors/search-box/connectSearchBox.js

```diff
@@ -151,6 +151,23 @@ export default function connectSearchBox(renderFn, unmountFn) {
         unmountFn();
         return state.setQuery('');
       },
+
+      getWidgetState(fullState, { state }) {
```

Comment:
According to the change in `getWidgetSearchParameters`:

- `fullState` should be renamed `uiState`
- `state` should be renamed `searchParam`